parsely-parsely-page.php

```diff
@@ -4,6 +4,9 @@
    <script type="application/ld+json">
    <?php echo json_encode($parselyPage); ?>
    </script>
+    <?php if (isset($parselyPage["custom_metadata"])) : ?>
+        <meta name='parsely-metadata' content='<?php echo json_encode($parselyPage['custom_metadata'])?>'>
+        <?php endif; ?>
```

Comment:
I see, and then `get_custom_metadata_for_post` would return values that they've set elsewhere by modifying their theme and adding inputs for those? Do you think it would be worth adding some documentation on this to explain how to configure this?